lib/ansible/modules/network/onyx/onyx_syslog_remote.py

```diff
@@ -37,7 +37,7 @@
     trap:
         description:
           - Minimum severity level for messages to this syslog server
-        choices: ['alert', 'crit', 'debug', 'emerg', 'err', 'info', 'none', 'notice', 'warning']
+        choices: ['none', 'info', 'notice', 'alert', 'debug', 'warning', 'err', 'emerg', 'crit']
```

Comment:
'debug' should be less than 'info'
please consult with onyx developers regarding the used enum